[2022-06-21 14:27:36,183] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_22 manual__2022-06-21T14:27:29.808157+00:00 [queued]>
[2022-06-21 14:27:36,200] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_22 manual__2022-06-21T14:27:29.808157+00:00 [queued]>
[2022-06-21 14:27:36,200] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-06-21 14:27:36,201] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-06-21 14:27:36,201] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-06-21 14:27:36,224] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): START_COUNT_BY_DAY_22> on 2022-06-21 14:27:29.808157+00:00
[2022-06-21 14:27:36,237] {standard_task_runner.py:52} INFO - Started process 2267 to run task
[2022-06-21 14:27:36,241] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'DM-COUNT-account-turnover', 'START_COUNT_BY_DAY_22', 'manual__2022-06-21T14:27:29.808157+00:00', '--job-id', '677', '--raw', '--subdir', 'DAGS_FOLDER/dm_fill_account_turnover.py', '--cfg-path', '/tmp/tmp9_ldu55e', '--error-file', '/tmp/tmp50m64w2d']
[2022-06-21 14:27:36,242] {standard_task_runner.py:77} INFO - Job 677: Subtask START_COUNT_BY_DAY_22
[2022-06-21 14:27:36,390] {logging_mixin.py:109} INFO - Running <TaskInstance: DM-COUNT-account-turnover.START_COUNT_BY_DAY_22 manual__2022-06-21T14:27:29.808157+00:00 [running]> on host c087b85f9f15
[2022-06-21 14:27:36,574] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=DM-COUNT-account-turnover
AIRFLOW_CTX_TASK_ID=START_COUNT_BY_DAY_22
AIRFLOW_CTX_EXECUTION_DATE=2022-06-21T14:27:29.808157+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-21T14:27:29.808157+00:00
[2022-06-21 14:27:36,609] {base.py:79} INFO - Using connection to: id: postgres. Host: postgres, Port: 5432, Schema: postgres, Login: ***, Password: ***, extra: {}
[2022-06-21 14:27:36,617] {dbapi.py:225} INFO - Running statement: CALL ds.fill_account_turnover_f (%(date)s::date);, parameters: {'date': '2018-01-22'}
[2022-06-21 14:27:37,260] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=DM-COUNT-account-turnover, task_id=START_COUNT_BY_DAY_22, execution_date=20220621T142729, start_date=20220621T142736, end_date=20220621T142737
[2022-06-21 14:27:37,313] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-21 14:27:37,887] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
